addons/mail/static/src/models/composer/composer.js

```diff
@@ -947,6 +959,10 @@ function factory(dependencies) {
         isLog: attr({
             default: false,
         }),
+        /**
+         * Determines whether the composer is actually posting a message.
```

Comment:
actually->currently

Also "composer is posting a message" is not very clear, maybe specify the request was sent to the server but its return is pending.